### DISCUSSION TITLE
How to enable anonymous Git read access for a repository if it does not appear in danger zone?

### DISCUSSION THREAD

**Discussion by motisaa** [community/community#72201] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hi, I just want to enable anonymous Git read access for a repository, reading the [document of github about this](https://docs.github.com/en/enterprise-server@3.7/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/enabling-anonymous-git-read-access-for-a-repository) I just can not find this option in the setting of the repository in the danger zone section. What should I do to enable it?

**Comment by Xrenya** [comment]


**Reply by motisaa** [reply]
It is public and not a fork and it is not disabled anonymous git read access. Just in the setting it does not appear to activate o disable it.

**Comment by motisaa** [comment, marked as answer]
I have understood that what I found was for "Enterprise Server," and what I wanted was for free account on GitHub. In fact, I wanted to know if it was possible to enable non-anonymous (i.e., restricted) read-only access in a repository on GitHub for a free account. And I have already discovered by [this github document](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/permission-levels-for-a-personal-account-repository#collaborator-access-for-a-repository-owned-by-a-personal-account) that it's not possible.
